src/pathfora.js

```diff
@@ -866,6 +895,53 @@
       return valid;
     },
 
+    entityFieldChecker: function (widget, fieldName, found) {
+      if (!found || !found.length) {
+        return true;
+      }
+
+      // for each template found...
+      for (var f = 0; f < found.length; f++) {
+        // parse the field name
+        var dataval = found[f].slice(2).slice(0, -2),
```

Comment:
this is fine but might be safer / support more complex things if we use regex here to define the parts rather than splits.